app/views/documents/new.html.erb

```diff
@@ -51,7 +50,8 @@
               <div class="flex-1"></div>
               <%= cp.text_field :overlap, value: default_chunk_overlap(default_chunking_method), size: 10, maxlength: 10, class: "rounded-md dark:bg-secondary-925 text-right" %>
             </div>
-          <% end %>
+            <% end%>
```

Comment:
```suggestion
            <% end %>
```

Reply:
Looks like some indentation might need an update too.

Reply:
In the original code the <% end %> doesn't match up with the form_for. I think rubocop and/or standard.rb can check erb files.  Rubocop does find some other minor fixes currently too.